modules/@apostrophecms/ui/ui/apos/components/AposButton.vue

```diff
@@ -71,6 +72,13 @@ export default {
     };
   },
   computed: {
+    role () {
+      if (this.type === 'group') {
+        return 'menuitem'
```

Comment:
I think it would be better to put the role attribute on the `<AposButton>` component than to tie it to a prop that's generally used for styling.